build_support/create-awsconfig.sh

```diff
@@ -0,0 +1,6 @@
+FILE=aws-exports.js
+if [ ! -f "./src/$FILE" ]; then
+  echo "$FILE does not exist in ./src/, creating aws-exports.js"
+  touch "./src/$FILE"
+  echo "export default {}" > "./src/$FILE"
```

Comment:
Add this line recommended by @josefaidt. With this addition it gets rid of a warning inside code editor saying that `aws-exports` wasn't a module